web/src/css/media_queries.scss

```diff
@@ -0,0 +1,52 @@
+/*
+Add media queries/breakpoints related rules here.
+*/
+@media screen and (max-width: 640px) {
+    .d-md-block {
+      display: block;
+    }
+    .d-md-none {
+      display: none;
+    }
+  
+    .subtitle {
+      flex-direction: column;
+      text-align: center;
+  
+      h5 {
+        line-height: 1.5rem;
+      }
+    }
+  
+    .mdc-data-table {
+      .mdc-data-table__header-cell,
+      .mdc-data-table__cell {
+        max-width: 100px;
+      }
+      .mdc-data-table__cell {
+        white-space: break-spaces;
+        line-height: 1.4;
+  
+        &.result {
+          text-align: right;
+        }
+      }
+      .embedded_region {
+        font-style: italic;
+        margin-top: 10px;
+      }
+    }
+}
+
+// Any rules for touch devices, can go here.
+@media (hover: none) {
+    #how-it-works-dialog-content {
+        a {
+            text-decoration: underline;
+        }
+    }
+
+    .mdc-data-table__sort-icon-button {
+        opacity: 0.2;
+    }
+}
```

Comment:
nit: newline

Reply:
> nit: newline

Fixed, but I think I'll soon add this in the eslint rules.

Reply:
Good idea. Also, there's an editor setting for this. I don't understand why it's not the default, but Linux tools expect it.